test/helpers.lua

```diff
@@ -575,20 +577,21 @@ local fixtbl_metatable = {
   end,
 }
 
-local function fixtbl(tbl)
+function module.fixtbl(tbl)
   return setmetatable(tbl, fixtbl_metatable)
 end
 
-local function fixtbl_rec(tbl)
+function module.fixtbl_rec(tbl)
+  local fixtbl_rec = module.fixtbl_rec
   for _, v in pairs(tbl) do
     if type(v) == 'table' then
       fixtbl_rec(v)
```

Comment:
why can't this (and the other cases) use `module.foo` ?

Reply:
I've done this in a few cases to minimize the diff / line length, and for performance (in loops).
Will revisit this then.